app/hosting.php

```diff
@@ -1,25 +0,0 @@
-<?php
-
-namespace Mangoweb;
-
-/**
- * @return bool
- */
-function isSharedHost() {
-	return strncmp(gethostname(), 'shared-', 7) === 0;
-}
-
-/**
- * @return bool
- */
-function isBetaHost() {
-	return strncmp(gethostname(), 'shared-beta', 11) === 0;
-}
-
-/**
- * @example 'shared-prod' for hostname 'shared-prod--i-i-78258ac5'
- * @return string
- */
-function getReplicationGroupName() {
-	return explode('--', gethostname())[0];
-}
```

Comment:
Smazani tohohle souboru nic nerozbije jo?

Reply:
Používalo se to výhradně pro nastavení Sentry (kontroloval jsem jestli se to ještě někde nepoužíva). Vzhledem k tomu, že to na kube nefunguje, tak pokud to někdo někdo používal, stejně to musí upravit.

Reply:
OK 👍